scripts/tokenomics-emulation/network.ts

```diff
@@ -226,6 +226,7 @@ class Network {
     res,
     rej,
     handleEvents,
+    params,
```

Comment:
No, it breaks other executions. You can pass params on the execution level.

Reply:
Done